icubam/backoffice/handlers/messages.py

```diff
@@ -18,9 +17,34 @@ def initialize(self):
     self.client = client.MessageServerClient(self.config)
 
   def prepare_for_table(self, msg):
-    msg['when'] = '{0:%Y/%m/%d at %H:%M:%S}'.format(
-      datetime.datetime.fromtimestamp(msg['when']))
-    return self.format_list_item(msg)
+    result = [
+      {
+        'key': 'user',
+        'value': msg['user_name'],
+        'link': f'{users.UserHandler.ROUTE}?id={msg["user_id"]}'
+      },
+      {
+        'key': 'ICU',
+        'value': msg["icu_name"],
+        'link': f'{icus.ICUHandler.ROUTE}?id={msg["icu_id"]}'
+      },
+    ]
+    msg_dict = {}
+    msg_dict['telephone'] = msg['phone']
+    msg_dict['scheduled'] ='{0:%Y/%m/%d at %H:%M:%S}'.format(
+        datetime.datetime.fromtimestamp(msg['when']))
+    msg_dict['attempts'] = msg['attempts']
+    msg_dict['first sent'] = 'not yet'
+    if msg['first_sent'] is not None:
```

Comment:
what's the difference bewteen "first_sent" and "first sent"? can we use the same key format?
(I feel like a key with space is a bit weird and error prone depending on how things get escaped etc)